rfcs/text/0001/process.yml

```diff
@@ -0,0 +1,13 @@
+---
+- name: process
```

Comment:
do these apply to `process.parent` asa well?

Reply:
Yes, now that `process.parent` is managed by the field reuse mechanism, this will indeed apply to it :-)